evap/evaluation/migrations/0143_alter_evaluation_state.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.0.4 on 2024-04-15 22:45
+
+import django_fsm
+import evap.evaluation.models
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("evaluation", "0142_alter_evaluation_state"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="evaluation",
+            name="state",
+            field=django_fsm.FSMIntegerField(
+                choices=[
+                    (evap.evaluation.models.Evaluation.State["NEW"], "new"),
```

Comment:
Is this what came out of `makemigrations`? Usually it would emit `10` instead of `Evaluation.State["NEW"]` here, with a good reason: The imported `Evaluation` class always reflects the current state of the codebase, so if we change `Evaluation.State` in the future, this migration would also use the changed version. We want the migration to make changes according to what `Evaluation.State` meant at the of writing though. See for example the migration `0123_evaluation_state_fsm_int` where I changed evaluation states to be numbers instead of strings. There, because the migration should work regardless of the current definition of `Evaluation`, all constants are copied in the migration